src/bp/core/config/config-loader.ts

```diff
@@ -184,7 +184,7 @@ export class ConfigProvider {
 
     // here it's ok to use the module resolver because we are discovering the built-in modules only
     const resolver = new ModuleResolver(this.logger)
-    return resolver.getModulesList().map(module => {
+    return (await resolver.getModulesList()).map(module => {
```

Comment:
was this broken ?

Reply:
As I said, it "worked" because of generators, but it shouldn't have been working. So that's why I'm a bit scared to just move forward with this change, until I check each .map to make sure that pattern wasn't used elsewhere :/